app/datasets/nypd_precincts.js

```diff
@@ -0,0 +1,695 @@
+export const nypdPrecinctLookup = [
```

Comment:
This is better but my issue is that this is a front end and data doesn't go in front ends (or at least, it's not supposed to). We'll cross this bridge if we decide this is worth releasing.